tests/functional/src/test/java/org/mule/functional/classloading/isolation/maven/dependencies/DependencyResolverTestCase.java

```diff
@@ -1,233 +0,0 @@
-/*
```

Comment:
My main concern about this PR is that is adding a lot of new complex code and there is no a single new unit test. We need tests for this runner, we can't depend on having to run all the isolated test in order to check that we did not break anything everytime we change this code